.github/workflows/master.yml

```
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Intel Corporation
+
+name: Master workflow
+on:
+  push:
+    branches:
+      - master
+      - GHA-integration
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+      - name: Build
+        run: |
+          go mod vendor
+          go build -mod=vendor -o build/_output/onos-config ./cmd/onos-config
+          rm -rf vendor
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+      - uses: golangci/golangci-lint-action@v4.0.0
+        with:
+          version: latest
+          args: -v --config ./.golangci.yml --timeout=15m
+
+  unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+      - name: Unit tests
+        run: go test -race github.com/onosproject/onos-config/...
```

I think this should only be this:
```suggestion
        run: go test -race ./...
```